modules/vaos/spec/requests/vaos/v2/locations/clinics_spec.rb

```diff
@@ -227,5 +227,53 @@
         end
       end
     end
+
+    describe 'GET recent clinics' do
+      let(:user) { build(:user, :vaos) }
+
+      context 'using VAOS' do
+        before do
+          Flipper.disable(:va_online_scheduling_use_vpg)
+        end
```

Comment:
If the results of the tests could be different with the feature enabled, then there needs to be a spec(s) for that 

```ruby
context 'without using VAOS' do
  before do
    Flipper.enable(:va_online_scheduling_use_vpg)
  end
```

Also consider changing the context description to:
`when va_online_scheduling_use_vpg is disabled (using VAOS)`

Reply:
We never expect the response data to be different based on the flipper, but the tests have been updated to reflect both cases